### PR TITLE
Add tests of `LaserScan2D`

### PR DESCRIPTION
**Pull request by H1rono**

### CODE REVIEW

File: arci/src/clients/dummy_laser_scan.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+use crate::{Error, LaserScan2D, Scan2D};
+
+#[derive(Debug, Clone, Default)]
+pub struct DummyLaserScan2D;
```
Comment (taiki-e) [inline arci/src/clients/dummy_laser_scan.rs:4]: Like other Dummy* types, we may want to store the value and allow it to be changed.
https://github.com/openrr/openrr/blob/8489d2c47339bcf9ee2879f1bc315a64489587fa/arci/src/clients/dummy_motor_drive.rs#L9-L11

Comment (OTL) [reply arci/src/clients/dummy_laser_scan.rs:4]: @H1rono Do you have any update about this?

Comment (H1rono) [reply arci/src/clients/dummy_laser_scan.rs:4]: Not yet. What kind of value we may want to change?

Comment (OTL) [reply arci/src/clients/dummy_laser_scan.rs:4]: Scan2D data which is returned by `fn current_scan()`.
(It might make some future tests easy)

**Review by taiki-e — APPROVED** [review]
Thanks!

**Pull request merged**